### PR TITLE
Update roundup email for 5OCT2018

### PR DESCRIPTION
**Pull request by mbeacom**
##### Description

The goal of this PR is to update the weekly roundup email copy and bounty references for the week of 5 October 2018.

##### Send-out Checklist

- [x] Check linting and ensure all strings are delimited/terminated properly (ensure no erroneous linting pops up with exception of line length)
- [x] Check on local via: http://localhost:8000/_administration/email/roundup
- [x] Validate all external and internal links are active/live
- [x] Have bizdev validate copy
- [ ] Merge, deploy live, and ensure all images, profiles, and bounties render in the email template
- [ ] Initiate sending out the roundup

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)

emails, roundup

##### Testing

Locally

### CODE REVIEW

File: app/retail/emails.py [hunk 1]
```diff
@@ -525,31 +525,30 @@ def render_start_work_applicant_expired(interest, bounty):
...
 </p>
 <p>
-If you're an OSS maintainer, we wrote a <a href="https://medium.com/gitcoin/a-maintainers-guide-to-hacktoberfest-21405c8ff09f">Guide to Hacktoberfest for maintainers!</a> It's
-a great look at how to manage an open source project generally, let us know what you think.
+Hacktoberfest is in full swing! Take a look here for issues tagged <a href="https://gitcoin.co/explorer?network=mainnet&keywords=hacktoberfest&order_by=-web3_created">'Hacktoberfest'</a>
+ which have been funded with Gitcoin bounties, across projects like Ruby For Good, Peepeth, Giveth and more.
 </p>
 <p>
-Lastly, October marks a month of travel for the Gitcoin team. If you'll be at ETH SF (Oct 5-7), Github Universe (Oct 16-17), Web 3 Summit (Oct 22 - 24), or Devcon 4, give us a shout!
+We're now on the move! If you'll be at ETH SF (Oct 5-7), Github Universe (Oct 16-17), Web 3 Summit (Oct 22 - 24), or Devcon 4, give us a shout!
```
[the remark this answers is not in the document]
Comment (owocki) [reply app/retail/emails.py]: can we add SustainOSS (Oct 25) to the list?

Comment (owocki) [inline app/retail/emails.py]: The Gitcoin Core team is on the move!

File: app/retail/emails.py [hunk 2]
```diff
@@ -525,31 +525,30 @@ def render_start_work_applicant_expired(interest, bounty):
...
 
     intro = '''
 
 <p>
 Hi there,
 </p>
 <p>
-<a href="https://medium.com/gitcoin/gitcoins-hacktoberfest-98825f199af2">Gitcoin's Hacktoberfest</a> starts on Monday! Post your favorite 3 open source projects on twitter & tag us (<a href="https://twitter.com/GetGitcoin">@GetGitcoin</a>), then make a <a href="https://gitcoin.co/requests">Gitcoin Request</a> to give those projects a boost!
+<a href="https://codefund.io/">CodeFund</a> wants to support open source projects everywhere!
+By placing ethical ads in beautiful places, Eric Berry and team have been providing Discourse, CodeSandbox, and Material-UI
```
Comment (owocki) [inline app/retail/emails.py]: `<a href="https://twitter.com/codeberry"> Eric Berry</a>`

File: app/retail/emails.py [hunk 3]
```diff
@@ -525,31 +525,30 @@ def render_start_work_applicant_expired(interest, bounty):
...
+and countless others with sources of revenue to support their projects. The CodeFund team just added a superstar, <a href="https://codefund.io/team">Nate Hopkins</a>,
+to the team as CTO, and we're excited about the future of the product. If you want to advertise to developers, there's no better way!
```
Comment (owocki) [inline app/retail/emails.py]: `<a href="https://codefund.io">Checkout CodeFund's new website to learn more</a>`

**Comment by mbeacom** [comment]
screenshot 2018-10-05 13 11 28

screenshot 2018-10-05 13 11 57

screenshot 2018-10-05 13 12 28

File: app/retail/emails.py (hunk 2 above)
Comment (owocki) [reply app/retail/emails.py]: coderberry not codeberry -- my bad

Comment (mbeacom) [reply app/retail/emails.py]: I didn't catch this either...  Sorry about that!

**Pull request merged**